nodes.py

```diff
@@ -871,6 +871,27 @@ def generate(self, width, height, batch_size=1):
         latent = torch.zeros([batch_size, 4, height // 8, width // 8])
         return ({"samples":latent}, )
 
+class NoisyLatentImage:
```

Comment:
I assume NoiseLatent is was added by mistake here from your other PR https://github.com/comfyanonymous/ComfyUI/pull/1099

Reply:
Oh my bad, removed NoisyLatentImage